.github/workflows/coverage.yml

```diff
@@ -0,0 +1,40 @@
+on: [push]
```

Comment:
Any reason to leave the brackets here and not in `ci.yml`?

Reply:
Nope